Layman-eUTxO-Design-Course/batch-1/stateanomaly/ergo-school-summary.md

```diff
@@ -0,0 +1,14 @@
+# Ergo School
```

Comment:
great. Can you add it into batch-1/teams/ergo-school/ergo-school-summary.md?

There should be a teams folder, or team.

Reply:
Done!